include/vara/Feature/Feature.h

```diff
@@ -274,4 +275,16 @@ class RootFeature : public Feature {
 
 } // namespace vara::feature
 
+inline std::ostream &operator<<(std::ostream &Out,
+                                const vara::feature::Feature *Feature) {
```

Comment:
```suggestion
                                const vara::feature::Feature &Feature) {
```
Should take the feature via `&` because we cannot handle a null state in the function and don't need to reassign the value.